Tests/L0/Maven/_suite.ts

```diff
@@ -209,12 +307,15 @@ describe('Maven Suite', function() {
                 assert(taskRunner.stdout.length > 0, 'should have written to stdout');
                 assert(taskRunner.failed, 'task should have failed');
 
-                assert(taskRunner.ran('/usr/local/bin/mvn -f pom.xml package jxr:jxr pmd:pmd'),
+                assert(taskRunner.ran('/usr/local/bin/mvn -f pom.xml package pmd:pmd -DlinkXRef=false'),
                     'should have run maven with the correct arguments');
 
                 done();
             })
             .fail((err) => {
+                console.log(taskRunner.stdout);
+                console.log(taskRunner.stderr);
+                console.log(err);
                 done(err);
```

Comment:
General test changes - only log stdout and stderr to the console if the task fails, to avoid cluttering up other people's test runs.